manifests/database.pp

```diff
@@ -14,9 +14,9 @@
   }
 
   include postgresql::server
-  postgresql::server::db { $pulpcore::database_name:
-    user     => $pulpcore::pulp_user,
-    password => postgresql_password($pulpcore::pulp_user, 'secret'),
+  postgresql::server::db { $pulpcore::postgresql_database_name:
+    user     => $pulpcore::user,
```

Comment:
For remote DB support we probably need a `postgresql_database_user` parameter. Fine to keep for a later PR though

Reply:
Opened issue #10 for this